R/RtGam.R

```diff
@@ -20,6 +21,13 @@
 #'   once.
 #' @param group The grouping variable for the case/reference-date pair. Not yet
 #'   implemented and a value other than `NULL` will throw an error.
+#' @param day_of_week A boolean or a vector of custom values to be applied to the
```

Comment:
Just flagging that this kind of complexity (arguments that change type) is an example of why I don't love the vector based interface we are working with here. 

I don't expect any action here so just close this out but perhaps we can think about extensibility in a later release (i.e what happens when we want this feature to be time-varying day of the week effects).

Reply:
The way this is currently phrased it sounds like you're trying to pass in a vector of coefficient values for the levels, which confused me on first read.

Reply:
```suggestion
#' @param day_of_week A boolean or a vector of custom levels to be applied to the
```